docs/apache-airflow/logging-monitoring/check-health.rst

```diff
@@ -72,6 +72,18 @@ status of the application. The return code is only indicative of the state of th
   monitoring, then in case of failure of the web server, you will lose the ability to monitor scheduler, which means
   that it can be restarted even if it is in good condition. For greater confidence, consider using :ref:`CLI Check for Scheduler <check-health/cli-checks-for-scheduler>`.
 
+.. _check-health/scheduler-health-check-server:
+
+Scheduler Health Check Server
```

Comment:
Can you update section "Health Check Endpoint" so that the user does not confuse the scheduler health control with the health control for the webserver? Now the user may think that the health check for scheduler is a subtype of the scheduler for the webserver, but that is a separate feature.